packages/integration-tests/projects/suite-web/tests/browser/outdated-chrome.test.ts

```diff
@@ -12,7 +12,8 @@ describe('Windows 7 with outdated chrome ', () => {
 
     it('Should just display outdated browser', () => {
         cy.prefixedVisit('/');
+        cy.document().its('fonts.status').should('equal', 'loaded');
```

Comment:
note to myself: I could move this one into 'prefixedVisit' command. but this can be done anytime later